mlkem/asm/asm.h

```diff
@@ -16,18 +16,60 @@ void intt_asm_opt(int16_t *);
 void poly_reduce_asm_clean(int16_t *);
 void poly_reduce_asm_opt(int16_t *);
 
+void poly_tomont_asm_clean(int16_t *);
+void poly_tomont_asm_opt(int16_t *);
+
+void poly_mulcache_compute_asm_clean(int16_t *, int16_t *, int16_t *, int16_t *);
+void poly_mulcache_compute_asm_opt(int16_t *, int16_t *, int16_t *, int16_t *);
+
+void poly_tobytes_asm_clean(uint8_t *r, const int16_t *a);
```

Comment:
What's the reasonf or not having an opt version of `poly_tobytes_asm_clean`?

Reply:
Some instructions are not yet supported by SLOTHY's uArch model. Let's address this in a follow-up.